kalite/__init__.py

```diff
@@ -6,13 +6,8 @@
 
 __version__ = VERSION
 
-
-# ROOT_DATA_PATH should point to the directory where the source files live, including the "docs" dir
-# TODO-BLOCKER(MCGallaspy): Use setuptools in the windows installer to avoid this nonsense.
-ROOT_DATA_PATH = os.environ.get(
-    "KALITE_ROOT_DATA_PATH",
-    os.path.join(sys.prefix, 'share', 'kalite')
-)
+# ROOT_DATA_PATH is *not* where the source files live. It's a place where non-user-writable data files may be written.
```

Comment:
Cool, so by this, we get rid of the environ KALITE_ROOT_DATA_PATH and can stop worry about yet another configurable gotcha :) :+1: